tests/e2e/standalone/standalone_test.go

```diff
@@ -587,6 +587,36 @@ func testList(t *testing.T) {
 		require.NoError(t, err, "dapr stop failed")
 		assert.Contains(t, output, "app stopped successfully: dapr_e2e_list")
 	}, "run", "--app-id", "dapr_e2e_list", "-H", "3555", "-G", "4555", "--", "bash", "-c", "sleep 10 ; exit 0")
+
+	t.Run("daprd instance in list", func(t *testing.T) {
+		homeDir, err := os.UserHomeDir()
+		require.NoError(t, err)
+
+		path := filepath.Join(homeDir, ".dapr")
+		binPath := filepath.Join(path, "bin")
+		daprdPath := filepath.Join(binPath, "daprd")
+
+		if runtime.GOOS == "windows" {
+			daprdPath += ".exe"
+		}
+
+		cmd := exec.Command(daprdPath, "--app-id", "daprd_e2e_list", "--dapr-http-port", "3555", "--dapr-grpc-port", "4555", "--app-port", "0")
+		cmd.Start()
+
+		output, err := spawn.Command(getDaprPath(), "list")
+		t.Log(output)
+		require.NoError(t, err, "dapr list failed with daprd instance")
+		listOutputCheck(t, output, false)
+
+		// TODO: remove this condition when `dapr stop` starts working for Windows.
+		if runtime.GOOS != "windows" {
+			output, err = spawn.Command(getDaprPath(), "stop", "--app-id", "daprd_e2e_list")
+			t.Log(output)
+			require.NoError(t, err, "dapr stop failed")
+			assert.Contains(t, output, "app stopped successfully: daprd_e2e_list")
+		}
+		cmd.Process.Kill()
```

Comment:
@shubham1172 please check stop works as expected here ??

Reply:
Does this needs a separate handling for windows ? If I understand it correctly from previous discussion then in windows `dapr stop` wont stop the daprd rt ?

Reply:
Yup, that is true. I will need to revert it. Just trying to confirm that this test actually fails on Windows (@shivamkm07 is helping me with testing).

Reply:
Thanks to @shivamkm07 for helping with Windows testing, the PR is no more a WIP.